ldk/javascript/src/filesystem/index.ts

```diff
@@ -134,6 +134,18 @@ export interface Filesystem {
    * @returns - the fully qualified path of the Loop's working directory as a string
    */
   workDir(): Promise<string>;
+
+  /**
+   * Get the list of retained file paths
```

Comment:
Do we need @ignore here? I'll go and approve this pr, you can add @ignore if you think you need to

Reply:
Good question! I'll check with Rich and come back to this. Thank you!